packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1191,6 +1191,23 @@ enum ResizeImagePolicy {
 /// Instructs Flutter to decode the image at the specified dimensions
 /// instead of at its native size.
 ///
+/// [width] and [height] parameters refer to logical pixels.
+///
+/// Logical pixels are roughly the same visual size across devices.
+/// Physical pixels are the size of the actual hardware pixels on the device.
+/// The number of physical pixels per logical pixel is described by the `devicePixelRatio`.
+///
+/// For example, if you want to resize the image to half of the screen's width, you can use:
+///
+/// ```dart
+///    Image(
+///      image: ResizeImage(
+///        FileImage(File(pathRefImage)),
```

Comment:
I would avoid a sample with an undeclared variable here. You never know if people copy the sample as a whole.
Also, I'm not sure if this sample would be ran through an analysis step / smoke test to see if it compiles.

```suggestion
///        FileImage(File('path/to/image')),
```

Reply:
> Also, I'm not sure if this sample would be ran through an analysis step / smoke test to see if it compiles.

In case it does, how should I proceed?
I didn't have the opportunity to face it yet.